addons/web_editor/static/src/js/wysiwyg/widgets/link_popover_widget.js

```diff
@@ -1,31 +1,85 @@
 /** @odoo-module **/
 
-import Widget from 'web.Widget';
-import {_t} from 'web.core';
-import {DropPrevious} from 'web.concurrency';
+import { _t } from "@web/core/l10n/translation";
 import { ancestors } from '@web_editor/js/common/wysiwyg_utils';
 
-const LinkPopoverWidget = Widget.extend({
-    template: 'wysiwyg.widgets.link.edit.tooltip',
-    events: {
-        'click .o_we_remove_link': '_onRemoveLinkClick',
-        'click .o_we_edit_link': '_onEditLinkClick',
-    },
-
+class DropPrevious {
```

Comment:
Meh, bad idea to duplicate IMO. Same as already explained but in this case, way worse that redefining a simple function like "confine".